gazebo/physics/ode/ODEPhysics.cc

```diff
@@ -1692,3 +1800,41 @@ bool ODEPhysics::GetParam(const std::string &_key, boost::any &_value) const
   }
   return true;
 }
+
+/// \brief A mathematical function that represents a saturation function
+/// that increases linearly until reaching a constant value with the addition
+/// of a deadband near zero. THe shape of this function for positive input
```

Comment:
capitalization: `THe` -> `The`

Reply:
https://github.com/osrf/gazebo/pull/3164/commits/cd9a5869193cbf59ba5e4b311856aa20668e0a20